lib/dfe/reference_data/itt/subjects.rb

```diff
@@ -667,7 +702,8 @@ module ITT
                                                         # FIXME: Counted as a language rather than "classics", is this correct?
```

Comment:
Not an issue for this PR, but worth raising a ticket and working with Rory/Elliott to confirm

Reply:
Ticket raised, currently in backlog